tests/entities/model_registry/test_registered_model.py

```diff
@@ -1,12 +1,10 @@
-import unittest
-
 from mlflow.entities.model_registry.model_version import ModelVersion
 from mlflow.entities.model_registry.registered_model import RegisteredModel
 from mlflow.entities.model_registry.registered_model_tag import RegisteredModelTag
 from tests.helper_functions import random_str
 
 
-class TestRegisteredModel(unittest.TestCase):
+class TestRegisteredModel:
```

Comment:
@viditjain99 Thanks for the PR! Can you convert methods to functions?

Example:

```python
class TestFoo:
    def test_foo(self):
        ...
```

to:

```python
def test_foo():
    ...
```

Reply:
sure

Reply:
I have made the required changes